src/hp/netcheck.rs

```diff
@@ -1822,17 +1822,39 @@ mod tests {
         let mut client = Client::new(None)
             .await
             .context("failed to create netcheck client")?;
-        let stun_port = 19302;
-        let host_name = "stun.l.google.com".into();
-        let derp_port = 0;
-        let mut dm = DerpMap::default_from_node(
-            host_name,
-            stun_port,
-            derp_port,
-            UseIpv4::None,
-            UseIpv6::None,
+
+        let stun_servers = vec![
+            ("stun.l.google.com", 19302, 0),
```

Comment:
can you anchor these at the root?  `stun.l.google.com.`?

Reply:
?

Reply:
because we do DNS lookups if you specify the stun server by hostname.  not having it anchored at the root means that the local "search" directives (i forget the official name) of your DNS configuration are applied.  e.g. on many home networks that would result in lookups to `stun.l.google.com.lan` or `stun.l.google.com.local`, as well as the intended `stun.l.google.com`.

By using `stun.l.google.com.` you avoid this.